### ISSUE TITLE
There is only one GPU and the weight problem of visiontransformer

### ISSUE THREAD

**Issue by Erik267** [pengzhiliang/MAE-pytorch#58]
Hello, I have three questions, I hope you can answer them.
I only have one GPU. When I use your model, I get an error, "not using distributed mode". What should I do? Heartfelt thanks.
Also, when running, it will also prompt 
```
"Weights of VisionTransformer not initialized from pretrained model: ['fc_norm.weight','fc_norm.bias','head.weight','head.bias']
Weights from pretrained model not used in VisionTransformer:.......
```

**Comment by pengzhiliang** [comment]
Actually, you can just set `--nproc_per_node=1`.

**Comment by Erik267** [comment]
Thanks very much for your answer. It is strange that I can successfully run through the visualization module by entering a picture.
But when fine-tuning, an error will still be reported,

```
load state dict by model_key = model
"Weights of VisionTransformer not initialized from pretrained model: ['fc_norm.weight','fc_norm.bias','head.weight','head.bias']
Weights from pretrained model not used in VisionTransformer
```

I sincerely hope that you can answer this question when you are free, thank you very much!

**Comment by pengzhiliang** [comment]
Ok, take it easy. 

In fact, it is normal because of the difference between pre-training model and fine-tuning model:
* keys `fc_norm.weight & fc_norm.bias` do not exist in the pretrianed model;
* the dim of `head.weight & head.bias`  are not the same.

And  the visualization just use pre-training model, so it's fully matched.
